plottr/data/datadict.py

```diff
@@ -618,7 +624,7 @@ def remove_unused_axes(self: T) -> T:
         """
         dependents = self.dependents()
         unused = []
-        ret = self.copy()
+        ret = self  #.copy()
```

Comment:
why do you self in a variable and return it, instead of just using self?

Reply:
fixed -- ready to approve the PR?